### ISSUE TITLE
GMM after epoch 200

### ISSUE THREAD

**Issue by aldakata** [bupt-ai-cz/PGDF#3]
After reading the paper: 
> Specifically, as shown in Figure 2, at each epoch, we get the clean probability wit of each sample from training loss by using Gaussian Mixture Model (GMM)

But on the code this GMM is only accounted for after epoch 200. Is this an error or am I missing something?
https://github.com/bupt-ai-cz/PGDF/blob/f7b6de71c0959c8a268a74da126a4e1858aeb290/train_cifar.py#L471

Thanks in advance!

**Comment by buptcwk** [comment]
Thanks for your question. In our implementation, we set "m=0" when "epoch <= 200 " because we think the "clean probability of prior knowledge (w_ip)" is more reliable before the classification model can work well and stably. So to accelerate the code, we did not calculate "w_it" when "epoch <= 200". This is a simple empirical parameter setting, maybe there is a better parameter setting scheme, such as making "m" gradually increase from 0.
